demo-spring-testing/src/main/java/com/examples/spring/demo/DemoSpringTestingApplication.java

```diff
@@ -7,6 +7,11 @@
 public class DemoSpringTestingApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(DemoSpringTestingApplication.class, args);
+		try {
+			SpringApplication.run(DemoSpringTestingApplication.class, args);
+		} catch (IllegalArgumentException e1) {
+		} catch (Throwable e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 12, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)